### ISSUE TITLE
Cannot find module '@comunica/actor-rdf-resolve-quad-pattern-file' when using a TriG file as data source

### ISSUE THREAD

**Issue by pietercolpaert** [comunica/comunica#120]
#### Issue type:

:bug: Bug

#### Description:

Executing `comunica-sparql https://linked.open.gent/parking 'SELECT * WHERE {<https://stad.gent/id/parking/P7> <http://vocab.datex.org/terms#parkingNumberOfVacantSpaces> ?free  . }'` returns an error: 

```bash
Error: Cannot find module '@comunica/actor-rdf-resolve-quad-pattern-file'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/pieter/.npm-global/lib/node_modules/@comunica/actor-init-sparql/engine-default.js:323:43)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
```

____
#### Environment:

v1.0.1

| software            | version(s)
| ------------------- | -------
| Comunica Init Actor | @comunica/actor-init-sparql 1.1
| node                | v9.4.0
| npm                 | 5.7.1
| Operating System    | Ubuntu

**Comment by RubenVerborgh** [comment]
Confirmed.

**Comment by joachimvh** [comment]
I think the problem stems from @comunica/actor-rdf-resolve-quad-pattern-file only being included in the devdependencies of the @comunica/actor-init-sparql package.json, causing it to be missing if you just to a normal install.

**Comment by pietercolpaert** [comment]
should I do a pull request where this is added in the dependencies? I acknowledge it actually works with a manual install after cloning the repo

**Comment by RubenVerborgh** [comment]
Because manual install also takes devdeps along. Should be in regular deps.
I wonder whether there are other packages in the wrong place?

**Comment by joachimvh** [comment]
That, or @comunica/actor-rdf-resolve-quad-pattern-file has to be thrown out of the default config. (The problem is that it tries to find the context for that package and doesn't find it because it's not installed.)

**Comment by rubensworks** [comment]
Yeah, that's my bad...

I indeed forgot to include it as an actual dependency in b780b07495d8bcbf0b642db015d059807ecc9358.

**Comment by rubensworks** [comment]
Fix released as 1.0.2.
